content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -6,6 +6,14 @@ weight: 100
 
 {{<glossary_definition term_id="node-pressure-eviction" length="short">}}</br>
 
+{{<note>}}
+The eviction signals, thresholds and metrics that were introduced as part of the
```

Comment:
This note reads pretty rough to me...

Also, the container runtime needs to implement this functionality. Currently this feature is only enabled for CRIO.

Reply:
Done.